tests/test_usm_utils.py

```diff
@@ -0,0 +1,42 @@
+import dpctl.tensor as dpt
+from numpy.testing import assert_equal
+
+import dpnp
+

```

Comment:
Could you please add `tests/third_party/cupy/misc_tests/test_byte_bounds.py` test scope based on CuPy?
Note, you can use `get_array()._pointer` instead of `data.ptr` there to make the tests working with dpnp.